src/clients/InsightsQueryClient.ts

```diff
@@ -534,6 +534,21 @@ class InsightsQueryClient {
     return response.data as Insights.InsightsQueryResponse<Insights.QueryGetFundingRateResponse>;
   }
 
+  async FundingHistoryGraphData(
+    query: Insights.QueryGetFundingHistoryGraphDataRequest
+  ): Promise<Insights.InsightsQueryResponse<Insights.QueryGetFundingHistoryGraphDataResponse>> {
+    const request = this.apiManager.path(
+      "market/funding/history/chart",
+      {},
+      {
+        market: query.market,
+        period: query.period,
+      }
+    );
+    const response = await request.get();
+    return response.data as Insights.InsightsQueryResponse<Insights.QueryGetFundingHistoryGraphDataResponse>;
+  }
```

Comment:
Pls add this query function to the [query_insights.ts](https://github.com/Switcheo/carbon-js-sdk/blob/feat/funding-view/examples/query_insights.ts) file.